### ISSUE TITLE
Implement GUI Library

### ISSUE THREAD

**Issue by jtheriault96** [Comet-Crusaders-Dev-Team/comet-crusaders#4]
**BLOCKED by #1**

### Problem
We need a better UI than simply reading and inputting text in the terminal. Ideally we should be able to let the player select options from menus, as well as dedicate a portion of the screen to show the status of the player's crew and ship. Setting some key commands to do things such as open crew/ship details and save the game would be nice to haves.

### Solution
Since there is nothing built into Java to do this, we'll need to use an external library. [Lanterna](https://github.com/mabe02/lanterna) seems like it would be able to cover most of our needs. Once it's been added to the project, Lanterna should be used in the `CharacterCreation` scenario, with choices from menus in place of all the code we use to prompt player input right now.

### Success Criteria
- [x] Lanterna added to Gradle.
- [ ] Have the game run within a Lanterna terminal instance, recreate the `CharacterCreation` scenario to use menus rather than take console input to allow players to make choices.
- [ ] Remove/refactor dead code that is no longer needed after the Lanterna migration.
